en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
-- `Yaw scale`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
+- `Yaw Torque`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
 - `Trim`: An offset added to the actuator so that it is centered without input.
   This might be determined by trial and error.
-- `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
+- (Advanced) `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
   - The setting limits the rate of change of an actuator (if not specified then no rate limit is applied).
      It is intended for actuators that may be damaged if they move too fast — such as the tilting actuators on a tiltrotor VTOL vehicle.
   - For example, a setting of 2.0 means that the motor/servo will not be commanded to move from 0 to 1 at a rate that completes the operation in less than 2 seconds (in case of reversible motors, the range is -1 to 1).
+- (Advanced) `Flap Scale`: How much this actuator is deflected at the "full flaps configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as flap or to compensate for generated torque through main flaps.
+- (Advanced) `Spoiler Scale`: How much this actuator is deflected at the "full spoiler configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as spoiler or to compensate for generated torque through main spoiler.
 - `Lock control surfaces in hover`:
   - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
   - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
 
+#### Flaps and Spoilers Configuration
+
+`Flap-control` and `spoiler-control` are aerodynamic configurations that can either be commanded manually by the pilot through a switch on the RC, or are set automatically by the controller, e.g. to reduce the airspeed before landing. 
+The flap and spoiler configuration controls are of the range of [0, 1] and can then be arbitrarily allocated to the control surfaces.
+
+It's important to understand the difference: with `flap/spoiler-control` we refer to an abstract metric of "how much flap/spoiler configuration" the controller currently demands, while the actual flaps or spoiler actuator setpoint can vary from that value.
```

Comment:
@sfuhrer I don't understand what you're saying here - the difference "to what".

I _think_ you might be trying to point out that the deflections indicate the maximum deflection that would be requested in the flap-control or spoiler-control configuration, BUT that this does not mean that this will be the actual deflection of the control surface. Instead, the deflection will be a combination of the requested deflection for flap/spoiler control and whatever other things are requested by the control loops for the control surface - such as yaw. 

Can you modify this to clarify, unless I'm just be thick?

Reply:
I expected that this was hard to understand.

What I want to say:
"flap control" is an abstract way of the controller to tell the allocator, from 0 to 1, how much it should adjust the aerodynamic property of the wings to the "full flaps control configuration". Then the allocator would can use all the available control surfaces it wants (usually flaps, ailerons, elevator). 

I hope this makes it a bit clearer, and if you'd have ideas of how to properly formulate/find fitting terms for that then let me know!

Reply:
I think so, but my suggestion above should tell you if I am or not: https://github.com/PX4/PX4-user_guide/pull/2450/files#r1170857584

I've tried to rewrite this bit below so it makes sense following my suggestion above.